pimd/pim_cmd.c

```diff
@@ -555,7 +555,7 @@ static void igmp_show_interfaces(struct pim_instance *pim, struct vty *vty,
 						: "down",
 					inet_ntop(AF_INET, &igmp->ifaddr, buf,
 						  sizeof(buf)),
-					pim_ifp->version,
+					pim_ifp->igmp_version,
```

Comment:
This can be modified to gm_version to be in line with what changes we are doing.

Reply:
No, it can't, because using the same field for IGMP and MLD version would be extremely confusing. An enum with values like `GM_IGMPV1` `GM_IGMPV2_MLDV1` `GM_IGMPV3_MLDV2` could _maybe_ work. Depends on what you're trying to do really.